core/src/Kokkos_View.hpp

```diff
@@ -2396,6 +2396,120 @@ KOKKOS_INLINE_FUNCTION DeducedCommonPropsType<Views...> common_view_alloc_prop(
   return DeducedCommonPropsType<Views...>(views...);
 }
 
+// -----------------------------------------------------------
+// public is_assignable
+// expose underlying view mapping properties case where layout isn't stride
+template <class DstViewType, class SrcViewType, class Enabled = void>
+struct is_always_assignable {
+  enum : bool { value = false };
+};
```

Comment:
Just inherit from `std::false_type`. That's the convention that's arising around these sorts of things.